.eslintrc

```diff
@@ -7,6 +7,7 @@
     },
     "extends": "airbnb",
     "rules": {
+        "max-len": "off",
```

Comment:
On our project, we have this config:
`"max-len": [2, {"code": 120, "tabWidth": 4, "ignoreUrls": true}],`